language/vm/vm-runtime/src/libra_vm.rs

```diff
@@ -605,6 +638,44 @@ impl VMExecutor for LibraVM {
     }
 }
 
+/// Internal APIs for the Libra VM, primarily used for testing.
+#[derive(Clone, Copy)]
+pub struct LibraVMInternals<'a>(&'a LibraVM);
```

Comment:
Just out of curiosity. Why do we need those wrappers since we can have those functions as public methods of LibraVM? Is it because we can add cfg flag easier in the future?

Reply:
Ooh a cfg flag is an interesting idea for the future! My immediate motivation was to just have a clear separation between the standard flow (validate_transaction, execute_block etc) and these sorts of uses.